doc/news/_2016-12-17_website_release.md

```diff
@@ -95,6 +95,11 @@ Clearly the service is meant to be driven by its users. Therefore we ask
 you to share your own configuration snippets, maybe they can be of help
 or simply a time saver for someone else!
 
+Snippets shared with the service are
+[BSD licensed](https://www.libelektra.org/devgettingstarted/license).
+The snippets can also be downloaded directly as bundle from a separate
```

Comment:
> I think that is clear

From which information should this be clear?

immediately -> shortly afterwards